bodhi/server/views/generic.py

```diff
@@ -221,7 +221,9 @@ def api_version(request):
 def notfound_view(context, request):
     """ Automatically redirects to slash-appended routes.
 
-    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#redirecting-to-slash-appended-rou
+    Note: The URL below spans two lines.
+    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html
+    #redirecting-to-slash-appended-rou
```

Comment:
It's worth noting that that fragment doesn't even exist. You _could_ just drop it.